### DISCUSSION TITLE
Welcome to rustdesk-api-server Discussions!

### DISCUSSION THREAD

**Discussion by kingmo888** [kingmo888/rustdesk-api-server#168] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AValov76** [comment]
Respect for the work done.
The API works great, I can connect to clients even from under RustDesk WebUI
If I log into the rustdesk-1.3.7-x86_64 application, I see a list in the address book
But! After logging into my account, I can no longer connect to any clients except myself. As soon as I log out of my account in rustdesk-1.3.7-x86_64, everything starts working.
Why is this so?


UPD
If you do not specify the –k switch (the one generated by the server will be used), then it will be impossible to work under the account.
**How can this be fixed?**

**Comment by rayangl** [comment]
<img width="651" height="428" alt="image" src="https://github.com/user-attachments/assets/e83d10d7-3cf9-4224-8608-97b3bf83eaa0" />
<img width="810" height="707" alt="image" src="https://github.com/user-attachments/assets/344aecd6-1ced-41eb-814d-d6d135f012f5" />
<img width="1667" height="930" alt="image" src="https://github.com/user-attachments/assets/c670d2ae-260e-4540-9cfe-5d1029b6ee3c" />
